javascripts/fetchIssueCount.js

```diff
@@ -1,5 +1,6 @@
 /* eslint global-require: "off" */
 /* eslint block-scoped-var: "off" */
+/* eslint prefer-arrow-callback: [ "error" ] */
```

Comment:
When I opened this issue I didn't expect it to cause these cascading issues, so I recommend adding these exclusions for now so we can get this change in:

```suggestion
/* eslint prefer-arrow-callback: [ "error" ] */
/* eslint arrow-parens: [ "error", "as-needed" ] */
/* eslint function-paren-newline: [ "off" ] */
/* eslint implicit-arrow-linebreak: [ "off" ] */
```